src/utils/getRandomWords.js

```diff
@@ -0,0 +1,5 @@
+// Function to get random words
+export const getRandomWords = (wordObjects, count = 4) => {
```

Comment:
Function name is descriptive enogh: getRandomWords And it is obvious that it is a function. No need to have a comment!